x/oracle/genesis.go

```diff
@@ -11,20 +11,21 @@ import (
 )
 
 func InitGenesis(ctx sdk.Context, keeper keeper.Keeper, data types.GenesisState) (res []abci.ValidatorUpdate) {
-
-	var wl []sdk.ValAddress
-	for _, addr := range data.AddressWhitelist {
-		if len(strings.TrimSpace(addr)) == 0 {
-			continue
+	if data.AddressWhitelist != nil {
+		wl := make([]sdk.ValAddress, len(data.AddressWhitelist))
+		for i, entry := range data.AddressWhitelist {
+			if len(strings.TrimSpace(entry)) == 0 {
+				continue
```

Comment:
This introduces a bug where wl has an zero value entry created by make() but not assigned here, which is why the code was using an unallocated slice.